modules/debts_api/lib/debts_api/v0/fsr_form_transform/enhanced_expense_calculator.rb

```diff
@@ -94,11 +98,14 @@ def default_expenses
         end
 
         def get_rent_mortgage_expenses
-          safe_number(@old_rent_mortgage_attr)
-          rent_or_mortgage_expenses = @expense_records.filter do |record|
-            [RENT, MORTGAGE_PAYMENT].include?(record['name'])
+          if @expense_page_active
+            safe_number(@new_rent_mortgage_attr)
+          else
+            rent_or_mortgage_expenses = @expense_records.filter do |record|
+              [RENT, MORTGAGE_PAYMENT].include?(record['name'])
+            end
+            safe_sum(rent_or_mortgage_expenses.pluck('amount')) + (@old_rent_mortgage_attr || 0)
```

Comment:
Would it make sense to do the ` || 0` in the initializer like other declarations and then we could remove it from here?

Reply:
More consistent, sounds good 👍